app/src/androidTest/java/com/github/palFinderTeam/palfinder/profile/FirebaseProfileServiceTest.kt

```diff
@@ -231,35 +232,35 @@ class FirebaseProfileServiceTest {
         assert(db.collection(PROFILE_COLL).document(id!!).get().await().toProfileUser()!!.achievements().isEmpty())
 
         firebaseProfileService.followUser(firebaseProfileService.fetch(id)!!, id2!!)
-        assert(db.collection(PROFILE_COLL).document(id).get().await().toProfileUser()!!.achievements().isEmpty())
-        assert(db.collection(PROFILE_COLL).document(id2).get().await().toProfileUser()!!.achievements().isEmpty())
+//        assert(db.collection(PROFILE_COLL).document(id).get().await().toProfileUser()!!.achievements().isEmpty())
+//        assert(db.collection(PROFILE_COLL).document(id2).get().await().toProfileUser()!!.achievements().isEmpty())
 
-        var list = List(4){" "}
+        var list = List(AchievementMilestones.MILESTONE1 - 1) { " " }
```

Comment:
difficult to understand the "-1" , but in test so not very urgent

Reply:
Yes we could use the `followCountAdapt` but since it's tests it was not a priority